### ISSUE TITLE
Compare Version Numbers - Microsoft

### ISSUE THREAD

**Issue by shelcia** [shelcia/noobcodes-docs#111]
https://leetcode.com/problems/compare-version-numbers

**Comment by Kumar-Ankit56** [comment]
@shelcia plz assign me.

**Comment by ankushj0801** [comment]
I want to  work on this please assign it to me if it is open. @shelcia

**Comment by ankushj0801** [comment]
I dont think anyone is working on it. Please assign it to me. @shelcia

**Comment by Kumar-Ankit56** [comment]
> I dont think anyone is working on it. Please assign it to me. @shelcia

Bro I'm on it

**Comment by shelcia** [comment]
anyone working ?

**Comment by Kumar-Ankit56** [comment]
> anyone working ?

Yes I'm on it.

**Comment by shelcia** [comment]
it has been 3 weeks ?!

**Comment by Kumar-Ankit56** [comment]
> it has been 3 weeks ?!

Yes whatever issue is assigned to me I will update soon from last 3 week I'm also not contribute in any repo extremely sorry for this.
My end sem is going on that's why.
